routes/index.js

```diff
@@ -38,22 +45,121 @@ router.get('/login', function (req, res, next) {
 //get breakfast menu page
 router.get('/breakfast', function (req, res, next) {
     Breakfast.find(function (err, docs) {
-        res.render('breakfast', {title: 'Breakfast Menu', breakfast: docs});
+        res.render('menus/breakfast', {title: 'Breakfast Menu', breakfast: docs});
+    });
+});
+
+router.get('/add-to-cart/breakfast/:id', function (req, res, next) {
+    let bfItemId = req.params.id;
```

Comment:
Best Practice: ```bfItemId``` is semantically poor. Does this stand for breakfastItemId? Consider that your variables should be easily identifiable to other devs viewing your code.

Reply:
@reactivepixel yes it does stand for breakfastItemId, I just didn't want the name to be too long, do you want me to change it?

Reply:
@reactivepixel fixed it

Reply:
Yup, Looking for descriptive variables. 

shr vrs cn b hrd 2 red

> short vars can be hard to read

And speed and accuracy both count in programming.